test/integrationtests/voight_kampff/features/steps/utterance_responses.py

```diff
@@ -65,6 +65,26 @@ def load_dialog_list(skill_path, dialog):
     return load_dialog_file(dialog_path), debug
 
 
+def _get_dialog_files(skill_path, lang):
+    """Generator expression returning all dialog files.
+
+    This includes both the 'locale' and the older style 'dialog' folder.
+
+    Args:
+        skill_path (str): skill root folder
+        lang (str): language code to check
+
+    yeilds:
+        (Path) path of each found dialog file
+    """
+    in_dialog_dir = Path(skill_path, 'dialog', lang).rglob('*.dialog')
+    for dialog_path in in_dialog_dir:
+        yield dialog_path
```

Comment:
should it search locale first or dialog first?

Reply:
It doesn't really matter as both should generally not exist in the same skill so I went alphabetically :)

Reply:
If you think it's more logical to do it in the reverse order I can update that.

Reply:
yeah, you're right, either seems fine